src/app/vm/vm-creation/template/agreement/vm-creation-agreement.component.ts

```diff
@@ -10,7 +10,13 @@ import { UserTagService } from '../../../../shared/services/tags/user-tag.servic
 
 @Component({
   selector: 'cs-vm-creation-template-agreement',
-  templateUrl: 'vm-creation-agreement.component.html'
+  templateUrl: 'vm-creation-agreement.component.html',
+  styles: [`
+    .agreement-wrapper {
```

Comment:
Move to separate file.
Let's do the template and styles inline or extracted from the file but not one inline and another not.